.travis.yml

```diff
@@ -6,30 +6,29 @@ services:
 language: java
 jdk: openjdk8
 
+env:
+  global:
+  - SOURCE_IMAGE_NAME="casesvc"
+  - DESTINATION_IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-casesvc"
+
 before_install:
-- cp .maven.settings.xml $HOME/.m2/settings.xml
-- mvn fmt:check
+  - cp .maven.settings.xml $HOME/.m2/settings.xml
+  - mvn fmt:check
 
 script: travis_wait mvn verify cobertura:cobertura-integration-test
 
 after_success:
-- if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
-  docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
-  docker push sdcplatform/casesvc;
+- if ([ "$TRAVIS_BRANCH" == "rm-census" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]) || ([ "$TRAVIS_BRANCH" == "automate-census-docker-build" ]); then
```

Comment:
need to remove the second part of this "if" as per the PR description before it can be merged